LION/models/CNNs/MSDNet.py

```diff
@@ -0,0 +1,288 @@
+# This file is part of LION library
+# License : BSD-3
+#
+# Author  : Charlie Shoebridge
+# Modifications: -
+# =============================================================================
+
+# Implementation of Mixed-Scale Dense Network described in:
+# Daniël M. Pelt and James A. Sethian
+# A mixed-scale dense convolutional neural network for image analysis
+
+import numpy as np
+import torch.nn as nn
+import torch
+from typing import Optional
+from LION.models.LIONmodel import LIONmodel, ModelInputType, ModelParams
+import warnings
+
+
+class MSD_Params(ModelParams):
```

Comment:
Humm you'll need to explain to me how having another class just for this helps, as I don't see the advantage

Reply:
In N2I for example, you need to supply a model that takes an image as input, since something special needs to be done with the reconstruction step (i e the averaging stuff) for example.

Reply:
Rather I think it's more important to have just MSDNet rather than just FBPMSD, with the former we still have the option to do either.